ray-operator/test/e2erayservice/support.go

```diff
@@ -154,11 +154,11 @@ func rayServiceSampleYamlApplyConfiguration() *rayv1ac.RayServiceSpecApplyConfig
 							WithResources(corev1ac.ResourceRequirements().
 								WithRequests(corev1.ResourceList{
 									corev1.ResourceCPU:    resource.MustParse("2"),
-									corev1.ResourceMemory: resource.MustParse("2Gi"),
+									corev1.ResourceMemory: resource.MustParse("3Gi"),
```

Comment:
I noticed that our ray serve app for this test is sometimes flaky due to the low memory limit:
<img width="1564" alt="Screenshot 2025-01-28 at 01 01 19" src="https://github.com/user-attachments/assets/d8e33a3a-7537-4e59-b18d-3dbb8a3433ce" />
It may require more than 2G of memory so I raise it to 3Gi.